src/include/platform/CHIPDeviceConfig.h

```diff
@@ -456,6 +456,15 @@
 #define CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF 0
 #endif
 
+/**
+ * CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
```

Comment:
Please don't put this in the middle of the PAF-related defines?

Reply:
Moved this to the end of the file